packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -675,6 +681,7 @@ export default class AwsS3Multipart extends UploaderPlugin {
         onComplete?.(etag)
         resolve({
           ETag: etag,
+          location,
```

Comment:
```suggestion
          uploadURL : location,
```

shouldn't it be uploadURL?

Reply:
Nope

Reply:
okidoki